### ISSUE TITLE
Possibly banned accounts are no longer marked red in "Manage accounts"

### ISSUE THREAD

**Issue by TFD777** [PokeAlert/PokeAlert#167]
I got a notification about few of my accounts being banned but they are still green on "Manage accounts" list.

**Comment by D3miS3raph** [comment]
The new version gives me a lot more connection issues in general with my accounts.  Especially with IV's turned on.  Have to stop and kill program and sometimes start a few times to get it working.  Sometimes the red circles don't disappear as well when this happens.  Very odd.  No issues in earlier releases.

**Comment by PokeAlert** [comment]
I slightly changed the scanning code which uses less memory. 
@TFD777 Are you certain it is the "Your account seems banned" message? and not the "login problem .. try in 1h" ?

**Comment by D3miS3raph** [comment]
I dont mind high memory use 😀

**Comment by PokeAlert** [comment]
Problem is that it makes the phone crash for several people

**Comment by D3miS3raph** [comment]
Ive had ivs on all evening and fine.  So not sure when and why it bugs but definitely an issue i had earlier.  Maybe just server load not sure.

Curious could you release a low and high memory version.  A low end path for people with weaker phones and high end for newer phones.

**Comment by TFD777** [comment]
@PokeAlert yes, it was try in 1h message, but doesn't it still mean that the account cannot be used? I think such accounts should be also marked in some color, for example in orange or yellow.

**Comment by PokeAlert** [comment]
In that case the message is wrong I just checked (copy past failed) that message appears when there was a connection problem (timeout,poor network or simply the calls failed because of some other reasons) 

I have it sometimes when I start a scan with over 20accs but if I retry directly after its all good

**Comment by WingChroniam** [comment]
I'm having the same issue.
